packages/camera/camera_android_camerax/lib/src/android_camera_camerax.dart

```diff
@@ -13,9 +18,67 @@ class AndroidCameraCameraX extends CameraPlatform {
     CameraPlatform.instance = AndroidCameraCameraX();
   }
 
+  ProcessCameraProvider? processCameraProvider;
+  CameraSelector? backCameraSelector;
+  CameraSelector? frontCameraSelector;
```

Comment:
I suggest either making these private and keeping the setters or keeping them public (and marking them `@visibleForTesting`) and setting them directly.

Reply:
(Optional suggestion) I'd suggest keeping the `CameraSelector`s public and remove the setters, but keep the `processCameraProvider` as is because if you move the null check to the setter for `processCameraProvider` like I commented below, it could be useful in this class, as well.